### PR TITLE
Calendar component fixes for IE and Edge

### PR DESCRIPTION
**Pull request by juliamaksimchik**
# What does this PR do:
 

-  Fixed [flex-basis bug](https://github.com/philipwalton/flexbugs#7-flex-basis-doesnt-account-for-box-sizingborder-box) in IE (workaround with width). 

-  Set `overflow:visible` prop for `button` to make pseudo-elements (before, after) visible in IE

-  Added a fix for IE and Edge to solve [border radius bug ](https://long2know.com/2015/09/ie11-rounded-corner-bug/) (white 1px line in calendar navigation)

-  `document.getElementById('output_2').value` is not working in IE, thus, replaces it with `document.getElementById('output_2').innerText`

# Where should the reviewer start:

    diff

Before in IE11:
ie-11-before

After in IE11:
ie-11-after

### CODE REVIEW

**Comment by AlleeX** [comment]
:muscle:

File: components/calendar/calendar.js [hunk 1]
```diff
@@ -213,15 +214,19 @@ export default class Calendar extends Component {
   }
 
   render() {
-    const { dataAttrs = {}, isDaySelectableFn, locale, navButtons, selectionType } = this.props;
+    const { dataAttrs = {}, isDaySelectableFn, locale, navButtons, selectionType, className } = this.props;
```
Comment (maximuk) [inline components/calendar/calendar.js]: default value for `dataAttrs` is not necessary, because we check it in `getDataAttributes` function

Comment (juliamaksimchik) [reply components/calendar/calendar.js]: Fixed

File: components/calendar/calendar.scss [hunk 2]
```diff
@@ -26,13 +26,19 @@
...
-  border-radius: var(--tx-calendar-nav-border-radius);
   display: flex;
   flex: 0 0 var(--tx-calendar-nav-height);
   flex-flow: row nowrap;
   width: 100%;
 }
 
+.ui-calendar-days__head {
+  display: flex;
+  flex-direction: column;
```
Comment (AntonTsymuk) [inline components/calendar/calendar.scss]: please check sorting

File: components/calendar/calendar.scss [hunk 3]
```diff
@@ -43,7 +49,8 @@
   box-shadow: 0 2px var(--tx-calendar-nav-button-box-shadow-color);
   cursor: pointer;
   color: var(--tx-calendar-nav-button-color);
-  flex: 0 0 var(--tx-calendar-nav-button-width);
+  flex: 0 0 auto;
+  width: var(--tx-calendar-nav-button-width);
   font-weight: var(--tx-calendar-nav-button-font-weight);
```
Comment (svetlanamorozko) [inline components/calendar/calendar.scss]: Please, check sorting

File: themes/_default.yaml [hunk 4]
```diff
@@ -201,9 +201,10 @@ button:
       decoration: "underline"
 calendar:
   height: 300px
+  head:
+    border-radius: "5px 5px 0 0"
```
Comment (iwwwi) [inline themes/_default.yaml]: before moving this variable here pls confirm that it is not overwritten in the travix-themes

Comment (juliamaksimchik) [reply themes/_default.yaml]: checked! It's not overwritten

Comment (iwwwi) [reply themes/_default.yaml]: if it is not used at all pls remove the variable and add it directly to the sass file as value... :)

**Pull request merged**